.gitignore

```diff
@@ -1,6 +1,5 @@
 node_modules/
 
-artifacts/
```

Comment:
I don't think we should remove `artifacts`. It's a directory created during a prepublish phase of `npm publish`.

Reply:
You're right. Restored [681438c](https://github.com/threshold-network/solidity-contracts/pull/143/commits/681438c1f3f758664beda91b74abfcc5bdca6339)